C23_WMDE_Mobile_DE_14

Abban opened the pull request:
- Based on the VAR of test 13
- VAR has a text test

Ticket: https://phabricator.wikimedia.org/T353635

gbirke commented on desktop_preact/Banner.jsx:
```diff
@@ -80,18 +82,24 @@ export default class Banner extends Component {
 				state.displayState === CLOSED ? 'wmde-banner--hidden' : '',
-				state.displayState === VISIBLE ? 'wmde-banner--visible' : ''
+				state.displayState === VISIBLE ? 'wmde-banner--visible' : '',
+				state.isAmountToggleOpen ? 'wmde-banner--amount-toggle-open' : '',
```
FYI, you can also provide objects with boolean values to toggle class names, e.g.
```javascript
{
   'wmde-banner--hidden': state.displayState === CLOSED, 
   'wmde-banner--visible': state.displayState === VISIBLE,
   'wmde-banner--amount-toggle-open': state.isAmountToggleOpen
}
```

This can help with the readability when all the classes come first and the conditions second and you don't have to provide empty strings in ternary operators. But ultimately it's a matter of opinion and taste and we mix and match the styles thoughout the code base.

gbirke commented on shared/components/ui/form/DonationForm.jsx:
```diff
@@ -50,6 +52,7 @@ export default function DonationForm( props ) {
 					currentValue={ paymentInterval }
 					onSelected={ e => setInterval( e.target.value ) }
 					disabledOptions={ disabledIntervals }
+					errorPosition={ props.errorPosition }
```
If this banner leaves the prototype stage we should definitvely document all the props (Like we do in `ProgressBar.js`). This is becoming confusing and I wouldn't know what value goes in here and where it comes from.

gbirke commented on shared/components/ui/form/SelectGroup.jsx:
```diff
@@ -2,7 +2,18 @@
 
-export default function SelectGroup( props ) {
+export const ERROR_POSITION_TOP = 1;
+export const ERROR_POSITION_BOTTOM = 2;
```
Good enough, but with potential for errors when we define the prop as `number`. Since this is an enum, we could potentialle use one of the following ES6 kludges to define it a bit more semantic/type-safe: https://stackoverflow.com/questions/44447847/enums-in-javascript-with-es6

Abban committed:
Archive C23_WMDE_Mobile_DE_13

Ticket: https://phabricator.wikimedia.org/T353635

Abban committed:
Prepare campaign for C23_WMDE_Mobile_DE_14

Based on the VAR of test 13

Ticket: https://phabricator.wikimedia.org/T353635

Abban committed:
Implement VAR for C23_WMDE_Mobile_DE_14

VAR has logos and logo buttons

Ticket: https://phabricator.wikimedia.org/T353635
